test/runtime/adapter-config.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var juttle_test_utils = require('./specs/juttle-test-utils');
+var check_juttle = juttle_test_utils.check_juttle;
+var path = require('path');
+var expect = require('chai').expect;
+var _ = require('underscore');
+var adapters = require('../../lib/runtime/adapters');
+
+
+function configure(name, conf) {
+    conf = _.clone(conf);
+    conf.path = path.resolve(__dirname, '../runtime/test-adapter-conf');
```

Comment:
Also I was thinking one better would be just to have:
```
var AdapterHelper = {
  register: function,
  configure:function
};
```
and then export that. That's better right?